frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -323,7 +324,7 @@ local footerTextGeneratorMap = {
         if prefix then
             string_percentage = prefix .. " " .. string_percentage
         end
-        return string_percentage:format(footer.progress_bar.percentage * 100)
+        return string_percentage:format(footer.pageno / footer.pages * 100)
```

Comment:
Looks like you may be losing the case & logic for/of the first branch here:
https://github.com/koreader/koreader/blob/f0f37e3153ad669ddcbcd1fa067533c99938ddb0/frontend/apps/reader/modules/readerfooter.lua#L2177-L2185

Reply:
Logic that I borrowed to use for the top status bar in https://github.com/koreader/koreader/commit/f00a88aef7e31c966e4e3e23e8dd3ccb54b0a2db

Reply:
Thanks